modules/postgres/vars.tf

```diff
@@ -0,0 +1,74 @@
+variable "identifier" {
+  description = "A unique name for the database"
+}
+
+variable "instance_class" {
+  description = "The size of the DB instances (ex: db.m4.medium, etc)"
+  type        = string
```

Comment:
Type string is a default, you don't need to specify it (in the same way that you are doing for the `identifier`)